### ISSUE TITLE
Blazor WASM - 56% size regression in dotnet.wasm.br

### ISSUE THREAD

**Issue by SamMonoRT** [dotnet/runtime#52389]
![image](https://user-images.githubusercontent.com/46026722/117314243-8d893f00-ae54-11eb-8745-15dacdad0c11.png)

cc @CoffeeFlux @lewing

**Comment by SamMonoRT** [comment]
@DrewScoggins - didn't see an auto-filled issue, last one is from 22hrs ago - not sure if there is any issue in that pipeline

**Comment by CoffeeFlux** [comment]
I'm not seeing any obvious candidates, so I'm going to bisect.

**Comment by SamMonoRT** [comment]
On looking closely at the powerbi tooltips  : Seems like an SDK change somewhere not sure how that relates to this : 6.0.100-Preview3.21168.18 to SDK verion 6.0.100-Preview5.21255.20

**Comment by CoffeeFlux** [comment]
A brotli's dotnet.wasm off main comes out to the expected (smaller) size:
```
-rwxr-xr-x   1 ryan  staff   778828 May  6 11:24 dotnet.wasm.br
```

It's likely that the relinking, which is available now with the SDK change, is somehow bringing in an additional library. cc: @lewing

**Comment by DrewScoggins** [comment]
There is something going wrong with the PowerBI.

Here are the actual data values in the DB, as you can see they do not show that 1.8MM number for the dotnet.wasm.br file. That is also why the auto-filer did not pick anything up, because we did not see a regression.
![image](https://user-images.githubusercontent.com/9439069/117339643-d214da80-ae54-11eb-999f-fa72c346ed76.png)

**Comment by DrewScoggins** [comment]
I am currently investigating why the PowerBI is doing this, but I believe that it is related to the fact that we have added new tests, and the dashboard was only designed for one test. We are probably getting some counters aggregated when we do not want it.

**Comment by lewing** [comment]
Just to be sure I tested relinking (with the workload installed) and it only helps. Definitely a dashboard thing,

**Comment by SamMonoRT** [comment]
Assigning to @DrewScoggins - please leave this open (even if there is an issue in the perf repo) till we have a fix for the dashboard to avoid re-opening of other similar issues.

**Comment by DrewScoggins** [comment]
It has been slow going on getting the dashboard fixed, but I should be pushing changes today that get everything back in working order.

**Comment by DrewScoggins** [comment]
@SamMonoRT PTAL at aka.ms/dotnetperfepic, to make sure that the changes I pushed have propagated. I see them on my end, but want to confirm that others are seeing this as well.

**Comment by SamMonoRT** [comment]
Yes, I see the right numbers/visual representation on the epic dashboard, But on https://msit.powerbi.com/groups/me/apps/54e0e83f-07bc-45bf-87b7-a7677ff3af2a/dashboards/fa051820-ff60-4d40-8a08-bdcc1b47b1d0 everything seems incorrect including the .NET5 baseline etc.
